src/vs/workbench/parts/debug/node/debugAdapter.ts

```diff
@@ -9,8 +9,9 @@ import * as strings from 'vs/base/common/strings';
 import * as objects from 'vs/base/common/objects';
 import * as paths from 'vs/base/common/paths';
 import * as platform from 'vs/base/common/platform';
+import fs = require('fs');
```

Comment:
I would put this at the top of the imports as we usually first put node imports than base, editor, platform, workbench